service/src/main/resources/application.yml

```diff
@@ -166,6 +166,7 @@ workspace:
         state: operating
 
   azure:
+    azureEnvironment: AZURE
```

Comment:
Is this necessary? There is already azure-environment on line 78 with the value `${env.azure.environment}` which refers to line 33 `environment: ${AZURE_ENVIRONMENT:AZURE}`

Reply:
Removed